extensions/wikia/PremiumPageHeader/PremiumPageHeaderController.class.php

```diff
@@ -354,6 +354,9 @@ private function getLanguages() {
 	}
 
 	private function getExplore(): array {
+		global $wgEnableDiscussions, $wgEnableForumExt;
+
+		//TODO: when productizing, please translate as crowdin messages or fix aliases of these special pages
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)